.vscode/settings.json

```
@@ -1,5 +1,6 @@
 {
     "typescript.tsserver.experimental.enableProjectDiagnostics": true,
     "cSpell.words": ["ecma", "popperjs", "postcondition", "Postcondition", "postconditions", "Svex", "threlte"],
-    "eslint.experimental.useFlatConfig": true
+    "eslint.useFlatConfig": true,
+    "typescript.tsdk": "node_modules\\typescript\\lib"
```

this path looks to be a windows specific as the filepath separator is using `\\`. is there a way to have these be dynamic to the current user's platform?

---

does setting it to `node_modules/typescript/lib` work for windows?

---

was able to get it to work on my machine! `/` is compatible with windows, so setting it to this should work for everyone:
```suggestion
    "typescript.tsdk": "./node_modules/typescript/lib"
```

---

Good catch, yes this does work!